guides/release/components/helper-functions.md

```diff
@@ -302,70 +299,114 @@ Class helpers are useful when the helper logic is fairly complicated, requires
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        Unlike nearly everything else in  template tag (`.gts`) files, these helpers do not need to be explicitly imported.
+      </div>
+    </div>
+    <img src="/images/mascots/zoey.png" role="presentation" alt="">
+  </div>
+</div>
 
 Below you will find some useful template helpers documented.
-For the full list of available helpers, you can check the [template helpers API documentation](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/).
+For the full list of available helpers, you can check the [template helpers API](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/) and [@ember/helpers](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper) documentation.
 
 ### The `get` helper
 
-The [`{{get}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/get?anchor=get)
+The [`{{get}}`](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper/methods/get?anchor=get)
 helper makes it easy to dynamically look up a property on an object or an element in an array. The second argument to `{{get}}` can be a string or a number, depending on the object being accessed.
 
-
 To access a property on an object with a string key:
 
-```handlebars
-{{get this.someObject "object_key"}}
+```gjs
+import { get } from '@ember/helper';
+
+const someObject = { object_key: 'Value' };
+
+<template>
+  {{get someObject "object_key"}}
+</template>
 ```
 
 To access the first element in an array:
 
-```handlebars
-{{get this.someArray 0}}
+```gjs
+import { get } from '@ember/helper';
+
+const someArray = [ 'one', 'two' ];
+
+<template>
+  {{get someArray 0}}
+</template>
 ```
 
 To access a property on an object with a dynamic key:
 
-```handlebars
-{{get this.address this.part}}
-```
+```gjs
+import { get } from '@ember/helper';
 
-If the `part` getter returns "zip", this will display the result of `this.address.zip`.
-If it returns "city", you get `this.address.city`.
+const address = { city: 'Chicago', state: 'IL', zip: '60610' };
+let part = 'zip';
 
+<template>
+  {{get address part}}
+</template>
+```
+
+If the `part` is "zip", this will display the result of `address.zip`.
+If it's "city", you get `address.city`.
 
 ### The `concat` helper
 
 We mentioned above that helpers can be nested. This can be
 combined with different dynamic helpers. For example, the
-[`{{concat}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/concat?anchor=concat)
+[`{{concat}}`](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper/methods/concat?anchor=concat)
 helper makes it easy to dynamically send a number of parameters to a component
 or helper as a single parameter in the format of a concatenated string.
 
-```handlebars
-{{get this.foo (concat "item" this.index)}}
+```gjs
```

Comment:
this is a weird example, I don't think we should involve `get` here.

(see my earlier note about encouraging using over all less `( )` in markup)